src/encoder/encoderffmpegcore.cpp

```diff
@@ -194,30 +196,31 @@ void EncoderFfmpegCore::encodeBuffer(const CSAMPLE *samples, const int size) {
 //
 // Currently this method is used before init() once to save artist, title and album
 //
-void EncoderFfmpegCore::updateMetaData(char* artist, char* title, char* album) {
+void EncoderFfmpegCore::updateMetaData(const QString& artist, const QString& title, const QString& album) {
     qDebug() << "ffmpegencodercore: UpdateMetadata: !" << artist << " - " << title <<
              " - " << album;
     m_strMetaDataTitle = title;
```

Comment:
Are these Info not used in the ffmpeg case? 
An other reason for an extra compile flag to disable ffmpeg for recording only